GVRf/Framework/src/org/gearvrf/GVRLight.java

```diff
@@ -12,27 +12,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package org.gearvrf;
 
-import static org.gearvrf.utility.Assert.*;
-
 import org.gearvrf.GVRMaterial.GVRShaderType;
+import org.gearvrf.utility.TextFile;
 
-public class GVRLight extends GVRHybridObject {
-
-    public GVRLight(GVRContext gvrContext) {
-        super(gvrContext, NativeLight.ctor());
-        // set light parameters to OpenGL default values
-        setPosition(0.0f, 0.0f, 1.0f);
-        setAmbientIntensity(0.0f, 0.0f, 0.0f, 1.0f);
-        setDiffuseIntensity(1.0f, 1.0f, 1.0f, 1.0f);
-        setSpecularIntensity(1.0f, 1.0f, 1.0f, 1.0f);
-        isEnabled = true;
+public class GVRLight extends GVRPointLight 
```

Comment:
GVRLight as we have it in GearVRf now is actually a point light implementation. In order to not change the existing API I left the name as it is. We really should just use GVRPointLight instead of GVRLight and deprecate the name GVRLight.